node/node.go

```diff
@@ -43,9 +43,9 @@ type Node struct {
 	// CoreClient provides access to a Core node process.
 	CoreClient core.Client `optional:"true"`
 
-	// RPCServer provides access to Node's exposed APIs.
-	RPCServer *rpc.Server `optional:"true"`
-
+	// rpc components
+	RPCServer  *rpc.Server  `optional:"true"`
+	RPCHandler *rpc.Handler `optional:"true"`
```

Comment:
nit: Handler is usually a specific handler for one specific endpoint. We have many of those, and calling them all as Handler feels a little bit odd to me, but that's a super nit. Also, I don't really see a reason to make it accessible from the Node, as it does not have any public fields.